man/docker-inspect.1.md

```diff
@@ -24,8 +24,85 @@ each result.
 **-f**, **--format**=""
     Format the output using the given go template.
 
+**--type**=*container*|*image*
```

Comment:
This flag also has to be added to the synopsis above; https://github.com/docker/docker/pull/13187/files#diff-49f81c25974e6fe4a56f32e78378faa2R10